src/pages/taesooKim/Login/Login.js

```diff
@@ -1,11 +1,86 @@
-import React, { Component } from "react";
+import React from "react";
+import { withRouter } from "react-router-dom";
+import "./Login.scss";
+import "../../../styles/common.scss";
+
+class LoginTaeSoo extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: "",
+      pw: "",
+      disabled: true,
+    };
+  }
+
+  activeLogin = () => {
+    this.state.id.indexOf("@") !== -1 && this.state.pw.length >= 5
+      ? this.setState({ disabled: false })
+      : this.setState({ disabled: true });
+  };
+
+  handleLogin = () => {
+    fetch("http://10.58.7.180:8000/user/signin", {
+      method: "POST",
+      body: JSON.stringify({
+        email: this.state.id,
+        password: this.state.pw,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => console.log(result));
+  };
+
+  handleInput = e => {
+    const { value, id } = e.target;
```

Comment:
```suggestion
    const { value, name } = e.target;
```
id는 앱 전체에 유니크한 셀렉터로서의 역할도 하고 있기 때문에 영향 없는 name이 좀 더 적절한 프로퍼티라고 생각되네요

Reply:
네네 수정하였습니다🐸